components/tile.scss

```diff
@@ -0,0 +1,7 @@
+$fd-icons-path: "../scss/icons/";
+
+@import "../scss/components/tile";
+@import "../scss/icons/icon";
+@import "../scss/components/image";
+@import "../scss/components/button";
+@import "../scss/components/identifier";
```

Comment:
This is another component that is a little strange to apply self-contained styles to because the majority of the style content loaded here is for components other than the tile.  I know these were added because of the playground examples, but honestly the playground examples should be reworked to _not_ include buttons, identifiers or images as those are completely self-contained themselves.  The playground examples can show a `<button>` element, but it shouldn't have any `fd-button` classes on it.  Similarly, images and identifiers can show an `<img>` or a `<div>` element, but those should not have any `fd-image` or `fd-identifier` classes.

On the main docs site, we can keep the buttons, images and identifiers in place as those are more use case-related.